src/components/NcActions/NcActions.vue

```diff
@@ -932,6 +932,40 @@ export default {
 }
 </style>
 ```
+
+#### Manually providing semantic menu information
+Due to limitations of Vue 2, when using a custom wrapper for action components, you have to provide the semantic menu type yourself.
```

Comment:
So the only solution would be to render hidden or shadow dom and then remove on next tick - just to check the children?

Reply:
To do the same what we do in `NcAppSidebarTabs` where component is not used as a component but as an object with data and actually renders nothing. But I'd say, it has too many disadvantages:
- Complexity. All `NcAction*` components have to be implemented in such a way.
- Old issue with order in Vue 2. Order of VNodes in slot is not the same as real order => reordering fails. In `NcAppSidebarTabs` it is not an issue because we have explicit `order` prop.
- Performance: rendering is slower. Even when menu is closed, on re-render we still have to render all (renderless) children, and register them just 

And it makes solution more and more hacky with not Vue-way approaches. We cannot guarantee it won't be broken after some minor upgrade with new reactivity or rendering optimization if we don't use it as it is supposed to be used.

Maybe there is a better solution, but I have no idea without huge a breaking change. Like stop using slot here at all and use props instead. Or use different components for different types and different slots for inline buttons and the menu.